### PR TITLE
(DPS-2910) Align Chat webhooks guide with webhooks v3.3

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/DPS-2910)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2910)

## 📓 Description
Since the way webhooks work changed significantly, we needed to adjust the Chat Webhooks guide to describe how to configure New Chat Webhooks.

### Content

- New content
- Proofreading/content fixes

### CODE REVIEW

File: content/getting-started/app-guides/chat-webhooks/index.mdx [hunk 1]
```diff
@@ -6,103 +6,116 @@ title: "Chat Webhooks"
...
+This document explains how to configure the **Chat Webhooks** building block in {{DEVELOPER_CONSOLE_URL}}. It was written for everyone aiming to configure webhooks in their integrations **via user interface** instead of calling the LiveChat [Configuration API](/management/configuration-api/#webhooks) directly. The document assumes you're already familiar with the general idea of webhooks.
 
-There are two ways to register webhooks for your application:
+The document applies to the so called **new chat webhooks** (v3.3). It's impossible to configure the old chat webhooks (v3.2 or below) via Developer Console – you can only delete the already configured ones. If you're curious about the differences between the old and new chat webhooks, [read this article] (https://developers.livechat.com/updates/setting-up-webhooks-with-livechat-api).
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:13]: Hm, but for now Dev Console will only support v3.3, right? For example, if you want to configure a webhook that exists in v3.4, but does not in v3.3, you will need to do that manually.

File: content/getting-started/app-guides/chat-webhooks/index.mdx [hunk 2]
```diff
@@ -6,103 +6,116 @@ title: "Chat Webhooks"
...
+#### Webhook triggers
 
-As a developer, you need to handle the authorization flow in your app. For a backend app, we recommend [Authorization code grant](/authorization/authorizing-api-calls/#authorization-code-grant), where your app exchanges `code` for `token`.
+Then, choose **webhook triggers**, the actions that will cause sending a webhook to the Webhook URL every time a specific event occurs. For the majority of events, you can also select a filter. In the example below, webhooks will be sent when there's a new chat or when someone sends new event to the chat. No filters are selected.
+
+- [The list of available webhooks](/management/configuration-api/#availabe-webhooks)
+- [The list of webhook filters]((/management/configuration-api/#triggering-actions))
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:102]: Yup, something went wrong here. I'll fix them.

File: content/management/configuration-api/index.mdx [hunk 3]
```diff
@@ -2420,7 +2420,13 @@ Here's what you need to know about **webhooks**:
...
+Registers a webhook for the Client ID (application) provided in the request. To receive webhooks, you need to enable them after registration. You can do that by calling the [Enable License Webhooks](#enable-license-webhooks) method that will install your registered webhooks for a given license. Keep in mind this two-step process applies to **license webhooks**.
+
+For **bot webhooks**, you need to [create a bot](#create-bot), [register webhooks](#register-webhook), and then, use [Set Routing Status](/messaging/agent-chat-api/#set-routing-status).
+
+As an alternative to calling Register Webhook, you can register webhooks via {{DEVELOPER_CONSOLE_URL}} by configuring the Chat Webhooks building block. In such a case, you don't need to give your app the `webhooks.configuration:rw` scope. Also, you won't have to support enabling webhooks as LiveChat will do that under the hood when someone installs your app. [Learn more...](/getting-started/app-guides/chat-webhooks/)
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/management/configuration-api/index.mdx:2427]: Almost got that right 😅
And what if a dev doesn't want to use a PAT, but a Bearer while following the manual flow? How would it look then? After all, we show a sample`register_webhooks` call with Bearer: https://developers.livechat.com/docs/management/configuration-api/#register-webhook

[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/management/configuration-api/index.mdx:2427]: And we specify `webhooks.configuration:rw` as the required scope (in all other cases in our docs, the **Required scopes** row refers to the application).

File: content/getting-started/app-guides/chat-webhooks/index.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (bkubiak) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:13]: Ok, I forgot that these section refers directly to the dev console

File: content/management/configuration-api/index.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (bkubiak) [reply content/management/configuration-api/index.mdx:2427]: discussed on slack, we decided that we want to suggest PATs in such cases (webhooks/props) and the app doesn't need these scopes

**Review by bkubiak — APPROVED** [review]

File: content/management/configuration-api/index.mdx [hunk 4]
```diff
@@ -2420,18 +2420,30 @@ Here's what you need to know about **webhooks**:
...
-|                 |                                                                          |
-| --------------- | ------------------------------------------------------------------------ |
-| Method URL      | `https://api.livechatinc.com/v3.3/configuration/action/register_webhook` |
-| Required scopes | `webhooks.configuration:rw`                                              |
+|                                        |                                                                          |
+| -------------------------------------- | ------------------------------------------------------------------------ |
+| Method URL                             | `https://api.livechatinc.com/v3.3/configuration/action/register_webhook` |
+| Required token scopes <sup>**1**</sup> | `webhooks.configuration:rw`                                              |
+
+**1)** Token scopes are assigned to an access token, not to an application (Client ID). For this method, we recommend authorizing with a [Personal Access Token (PAT)](/authorization/authorizing-api-calls/#personal-access-tokens). Make sure, your PAT has the `webhooks.configuration:rw` scope.
```
Comment (oliwiapolec) [inline content/management/configuration-api/index.mdx:2440]:
```suggestion
**1)** Token scopes are assigned to an access token, not to an application (Client ID). For this method, we recommend authorizing with a [Personal Access Token (PAT)](/authorization/authorizing-api-calls/#personal-access-tokens). Make sure your PAT has the `webhooks.configuration:rw` scope.
```

File: content/getting-started/app-guides/chat-webhooks/index.mdx [hunk 5]
```diff
@@ -6,103 +6,118 @@ title: "Chat Webhooks"
...
 
-There are two ways to register webhooks for your application:
+The document applies to the so called **new chat webhooks** (v3.3). It's impossible to configure the old chat webhooks (v3.2 or below) via Developer Console – you can only delete the already configured ones. If you're curious about the differences between the old and new chat webhooks, [read this article](https://developers.livechat.com/updates/setting-up-webhooks-with-livechat-api).
 
-1. **Directly through the Configuration API.** To do that, call the [Register Webhook](/management/configuration-api/v3.2/#register-webhook) method with the desired configuration.
+Keep in mind that the Chat Webhooks building block allow you to configure `license` webhooks, and that is what this document focuses on. The document doesn't apply to `bot` webhooks.
```
[the remark this answers is not in the document]
Comment (oliwiapolec) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:15]: shouldn't we add the articles here? :D

[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:15]: Yes, we should :)

File: content/getting-started/app-guides/chat-webhooks/index.mdx [hunk 6]
```diff
@@ -6,103 +6,118 @@ title: "Chat Webhooks"
...
-### Frontend apps
+### Before you start
 
-In a nutshell:
+Log in to {{DEVELOPER_CONSOLE_URL}} or create an account for free.
 
-1. Create an app in {{DEVELOPER_CONSOLE_URL}} and add the Agent App Widgets building block.
-2. Set up the App Authorization building block and implement an authorization flow in your app.
-3. Configure the Chat webhooks building block.
+### Step 1 Create an app in Developer Console
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:37]: It does, but it will break links :/

File: content/getting-started/app-guides/chat-webhooks/index.mdx [hunk 7]
```diff
@@ -6,103 +6,118 @@ title: "Chat Webhooks"
...
+- Provide a **Direct installation URL** and add it to the Redirect URI whitelist.
 
-![Chat Webhooks LiveChat](./chat-webhooks-block-2x-livechat.png)
+💡 If you don't know what it is and how to use it, watch our live coding session where we expained the topic step by step.
 
-### Backend apps
+**Handle direct installation and authorization in your app**
 
-In a nutshell:
+You're building a backend app – it has no frontend side, no widget, no interface to communicate with the user. How's the user supposed to authorize your app if there's nowhere to click? The answer is: direct installation flow.
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:75]: Good intuition 🤓

File: content/getting-started/app-guides/chat-webhooks/index.mdx [hunk 8]
```diff
@@ -6,103 +6,118 @@ title: "Chat Webhooks"
...
 
-Usually, the authorization process starts in the Agent Application when a user runs the app for the first time. With the direct installation flow enabled, authorization begins on the Marketplace. When a user installs your app, a pop-up with the LiveChat authorization flow appears, and the application asks for access to certain resources. Upon successful authorization, the user is redirected to the **Direct installation URL**, where they can, for example, do the initial app configuration. (It all depends on what you implemented.)
+#### Webhook triggers
 
-As a developer, you need to handle the authorization flow in your app. For a backend app, we recommend [Authorization code grant](/authorization/authorizing-api-calls/#authorization-code-grant), where your app exchanges `code` for `token`.
+Then, choose **webhook triggers**, the actions that will cause sending a webhook to the Webhook URL every time a specific event occurs. For the majority of events, you can also select a filter. In the example below, webhooks will be sent when there's a new chat or when someone sends new event to the chat. No filters are selected.
```
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:101]: Yes, there was supposed to be an image with a sample configuration but I'm waiting for the UI to be coded to take a screenshot. Good point though, I almost forgot about it!

File: content/getting-started/app-guides/chat-webhooks/index.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (oliwiapolec) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:37]: Ohhh aight :(

File: content/getting-started/app-guides/chat-webhooks/index.mdx [hunk 9]
```diff
@@ -3,106 +3,125 @@ weight: 70
...
+**Configure the App Authorization building block**
 
-To finish off, install the app privately (**Private installation** tab in Developer Console) and perform an action that's your webhook trigger. You can also ensure the webhooks were registered properly by calling [List Registered Webhooks](/management/configuration-api/v3.2/#list-registered-webhooks). Remember about adding the right scopes to use those methods.
+- Add the `webhooks.state:rw` scope, which will be later on used by LiveChat to enable webhooks for a license that installs your app. (If you don't add this scope, Developer Console won't allow for creating the Chat Webhooks block.)
+- Provide a **Direct installation URL** and add it to the Redirect URI whitelist.
 
-![Chat Webhooks LiveChat](./chat-webhooks-block-2x-livechat.png)
+💡 If you don't know what it is and how to use it, watch our [live coding session][(https://www.youtube.com/watch?v=ei8YG9mLdfI) where we explained the topic step by step.
```
Comment (oliwiapolec) [inline content/getting-started/app-guides/chat-webhooks/index.mdx:71]:
```suggestion
💡 If you don't know what it is and how to use it, watch our [live coding session](https://www.youtube.com/watch?v=ei8YG9mLdfI) where we explained the topic step by step.
```

Comment (oliwiapolec) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:71]: :D

**Review by oliwiapolec — APPROVED** [review]

File: content/getting-started/app-guides/chat-webhooks/index.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply content/getting-started/app-guides/chat-webhooks/index.mdx:37]: I lied to you. We used to have a problem with that, but apparently it works now. So I'll change it :)

**Comment by ale-kacperczyk** [comment]
Duplicated
